//package com.dev.exceptions;
//
//public class ExceptionHandling2 {
//
//	public static void main(String[] args) throws CustomException {
//	try {
//		s();
//		System.out.println("No exception for s()");
//		int res  = add(10,0o)
//	}
//
//	}
//
//}
